lib/cli-config.js

```diff
@@ -68,14 +71,34 @@ exports.getContent = function(config, directory) {
         config = path.basename(config);
         ext = path.extname(configPath);
 
-        if (ext === '.js' || ext === '.json') {
```

Comment:
FYI: I moved the JSON down the the final else statement, so that we can benefit from jsonlint's parser.